java/ql/src/experimental/semmle/code/java/frameworks/FreeMarker.qll

```diff
@@ -0,0 +1,29 @@
+/** Definitions related to the FreeMarker Templating library. */
+
+import semmle.code.java.Type
```

Comment:
Conventionally, simply `import java` unless you need things that aren't in the default java namespace (`Class` and `Method` are)